README.md

```diff
@@ -54,10 +54,16 @@ def my_function(sender, instance, **kwargs):
 python manage.py process_future_tasks
 ```
 
+**Command for starting the periodic future task processing**
+```bash
+python manage.py populate_periodic_future_tasks
+```
+
 ## Django Compatibility Matrix
 
 If your project uses an older verison of Django or Django Rest Framework, you can choose an older version of this project.
 
 | This Project | Python Version       | Django Version |
 |--------------|----------------------|----------------|
+| 1.1.*        | 3.8, 3.9, 3.10, 3.11 | 3.2, 4.0, 4.1  |
```

Comment:
We should support only `3.2`, `4.1` and `4.2` here, as 4.0 is EOL - https://endoflife.date/django

Reply:
True, I have changed this now. Thank you!